README.md

```diff
@@ -100,335 +154,353 @@ immunopepper make_bg \
+    One example of how the output name for this file could look is the following: *breast_TCGA-OR-A5J1-01A-11R-A29R-07_somatic_SampleLim3.0_CohortLim0.0Across10_FiltNormalsGtexcoreCohort_ExceptTCGA-OR-A5J1-01A-11R-A29R-07_CohortLim0.0_Across1_FiltUniprot_batch0_1.tsv.gz*.
+
+   #TODO: Check if the output is still like this
+    ````
+    kmer    TCGAA2A0SX01A12RA08407all       kmer_IL_eq
+    AAGDDENHN       244.0   AAGDDENHN
+    AAPGQHLQA       38.0    AAPGQHLQA
+    ACSNFIFKH       114.0   ACSNFLFKH
+    ````
+    3. **{`--output-count`}**: If `--output-count` is provided, this file will be generated. It contains the number of remaining kmers after each filtering step. It is a tabular file with different fields:
+    - Sample: The cancer sample id. It is an id matching `--ids-cancer-samples`.
+    - Mutation mode: The mutation mode used for the analysis. It is either `ref`, `somatic`, `germline` or `somatic_and_germline`.
+    - Min_sample_reads: Expression threshold for sample filtering (step 2a of cancer pipeline). It is the threshold provided under `--sample-expr-support-cancer`.
+    - Number (#) of cohort samples: Number of samples threshold for cancer cohort filtering (step 2b of cancer pipeline). It is the threshold provided under `--n-samples-lim-cancer`.
+    - Reads per cohort sample: Expression threshold for expression cancer cohort filtering (step 2b of cancer pipeline). It is the threshold provided under `--cohort-expr-support-cancer`.
+    - Number (#) of normal samples allowed: Number of samples threshold for normal cohort filtering (step 2b of normal pipeline). It is the threshold provided under `--n-samples-lim-normal`.
+    - Normal cohort id: The normal cohort id.
+    - Reads per normal sample: Expression threshold for expression normal cohort filtering (step 2a of normal pipeline). It is the threshold provided under `--cohort-expr-support-normal`.
+    - Init_cancer: Number of kmers before any filtering step.
+    - Filter_sample: Number of kmers remaining after sample filtering (step 2a of cancer pipeline).
+    - Filter_sample_cohort: Number of kmers remaining after cancer cohort filtering (step 2b of cancer pipeline).
+    - Filter_sample_cohort_cohortnormal #TODO: check if this was corrected: Numberof kmers after differential filtering with normal samples
```

Comment:
Filter_sample_cohort_cohortnormal 
- Is there a hyphen missing Filter_sample_cohort_cohort_normal 
- Yes this is the number after differential filtering of the normal samples. See explanation below:

kmers included in the normal cohort must pass the number of samples and expression threshold for the normals `--n-samples-lim-normal `and `cohort-expr-support-normal` respectively. This means that the cancer-specific kmers will be below these thresholds in the normal cohort)

Reply:
I saw in the code for cancer specif mode that you save it as 'Filter_Sample_Cohort_CohortNormal' (line 249)